include/z64.h

```diff
@@ -184,10 +184,22 @@ typedef struct {
     /* 0x0104 */ Vec3f  curDistortionOrientation;
     /* 0x0110 */ Vec3f  curDistortionScale;
     /* 0x011C */ u16    normal; // used to normalize the projection matrix
-    /* 0x0120 */ s32    flags;
+    /* 0x0120 */ s32    dirtyFlags;
     /* 0x0124 */ s32    unk_124;
 } View; // size = 0x128
 
+#define VIEW_VIEWING (1 << 0)
+#define VIEW_VIEWPORT (1 << 1)
+#define VIEW_PROJECTION_PERSPECTIVE (1 << 2)
+#define VIEW_PROJECTION_ORTHO (1 << 3)
+#define VIEW_ALL (VIEW_VIEWING | VIEW_VIEWPORT | VIEW_PROJECTION_PERSPECTIVE | VIEW_PROJECTION_ORTHO)
+
+#define VIEW_FORCE_VIEWING (VIEW_VIEWING << 4)
+#define VIEW_FORCE_VIEWPORT (VIEW_VIEWPORT << 4)
+#define VIEW_FORCE_PROJECTION_PERSPECTIVE (VIEW_PROJECTION_PERSPECTIVE << 4)
+#define VIEW_FORCE_PROJECTION_ORTHO (VIEW_PROJECTION_ORTHO << 4)
+#define VIEW_FORCE_ALL (VIEW_FORCE_VIEWING | VIEW_FORCE_VIEWPORT | VIEW_FORCE_PROJECTION_PERSPECTIVE | VIEW_FORCE_PROJECTION_ORTHO)
```

Comment:
this define doesn't make sense, using both of `VIEW_FORCE_PROJECTION_PERSPECTIVE | VIEW_FORCE_PROJECTION_ORTHO` is contradictory

maybe `#define VIEW_FORCE_ALL_PERSPECTIVE (VIEW_FORCE_VIEWING | VIEW_FORCE_VIEWPORT | VIEW_FORCE_PROJECTION_PERSPECTIVE)` would be useful though. it could replace some `|`s. But I'm not a fan of how "all_perspective" looks

Reply:
You're right, the "force all" mask doesn't make any sense. I've removed it. Would be nice to cut down on those `|`s too, though maybe not such a pressing issue.

a2e31a282cf4a47c09ede9eebd1adf74f0be015a